src/TBuffer.h

```diff
@@ -132,12 +133,12 @@ enum TMXPMode
 
 class TBuffer
 {
-    // need to use tr() on encoding names in csmEncodingTable
-    Q_DECLARE_TR_FUNCTIONS(TBuffer)
-
     // private - a map of computer-friendly encoding names as keys,
-    // values are a pair of human-friendly name + encoding data
-    static const QMap<QString, QPair<QString, QVector<QChar>>> csmEncodingTable;
+    // value is the encoding data.
+    // Formerly a friendly name was stored here but that was subjected to
```

Comment:
History is better for Git.

Reply:
I felt I had to explain what was going on - and why it had changed - and where to find the missing details now.

I will look to see if I can reword it...